apps/gitness/src/pages-v2/repo/repo-summary.tsx

```diff
@@ -29,6 +27,7 @@ import {
 } from '@harnessio/ui/views'
 import { SummaryItemType } from '@harnessio/views'
 
+import { BranchSelectorContainer } from '../../components-v2/branch-selector-container'
```

Comment:
```suggestion
import { BranchSelectorContainer } from '@/components-v2/branch-selector-container'
```

makes the imports indepenent of filepath

Reply:
I don't think we have aliases in gitness.